lib/Twig/Extensions/Template/DisplayCallGraphTemplate.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/*
+ * This file is part of Twig.
```

Comment:
It's the same sentence for all files in this repository. For instance: https://github.com/twigphp/Twig-extensions/blob/master/lib/Twig/Extensions/Autoloader.php